### PR TITLE
rectified the installation commands for various platform (Issue : #502) 

### PR DESCRIPTION
**Pull request by shahkushal38**
**Description**

This PR fixes #
rectifying the installation commands for various platform #502 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by warunicorn19** [comment]
Why have you made changes to the Gemfile?

**Comment by shahkushal38** [comment]
> Why have you made changes to the Gemfile?

Actually I was facing problem with the version of ruby so had to make some changes there in order to make site run in my local system

**Comment by warunicorn19** [comment]
Ohh okay, but you cannot commit those changes. Could you please revert the change back to what it was?

**Comment by samirparhi-dev** [comment]
as @warunicorn19  said , Please do not commit your env specific code :-) Please revert, All other changes LGTM.

**Comment by shahkushal38** [comment]
> Ohh okay, but you cannot commit those changes. Could you please revert the change back to what it was?
Ohh okay, I think I have done that it's just a new line that has been added by mistake to gemfile, should I make a new PR in that case?

**Comment by warunicorn19** [comment]
You could just change it back and make a commit to this PR itself, that should be fine

**Comment by shahkushal38** [comment]
> You could just change it back and make a commit to this PR itself, that should be fine

Yes,I have done that with a new commit and restored the gemfile to what it was previously. Does this work now?

File: _includes/getting-started.html [hunk 1]
```diff
@@ -318,24 +304,27 @@ <h3>Helm Chart:</h3>
...
                 <a class="btn tooltip" style="position: absolute; top:0.10rem; right: 0.5rem; font-size: 22px;" data-clipboard-target="#start-helm-v3"
                   data-clipboard-text="kubectl create namespace meshery
-                  helm install meshery --namespace meshery install/kubernetes/helm/meshery" onmouseout="resetCopyText(this)">
+                helm repo add meshery https://meshery.io/charts/
+                helm install meshery meshery/meshery -n meshery" onmouseout="resetCopyText(this)">
                   <i class="far fa-copy"></i>
                   <span class="tooltiptext" style="font-size: 15px; width: 140px; height: 40px; padding: 0; line-height: 40px; top: 3rem; left: -20px;">Copy to clipboard</span>
                 </a>
             </div>
             <p style="font-size: 1em;">Not a Helm user? Use the Meshery manifests directly.</p>
```
Comment (Nikhil-Ladha) [inline _includes/getting-started.html:320]: If we are not showing the `manifests` command then, why show this line?

Comment (samirparhi-dev) [reply _includes/getting-started.html:320]: That is for FYI to the end user.

**Review by leecalcote — CHANGES_REQUESTED** [review]
@shahkushal38 thanks for this. Will you take a moment to remove the changes made to the gemfile and gemfile.lock?

**Comment by shahkushal38** [comment]
> @shahkushal38 thanks for this. Will you take a moment to remove the changes made to the gemfile and gemfile.lock?

Yes @leecalcote , I have done that in my last commit, can you please check whether the required changes have been made or else I redo it.

**Comment by leecalcote** [comment]
> > @shahkushal38 thanks for this. Will you take a moment to remove the changes made to the gemfile and gemfile.lock?
> 
> Yes @leecalcote , I have done that in my last commit, can you please check whether the required changes have been made or else I redo it.

Thank you. I still see both of these files included. Despite not having any material changes, they shouldn't be included at all. Thanks for having a look at this.

File: charts/index.yaml [hunk 2]
```diff
@@ -1,72 +1,6 @@
 apiVersion: v1
 entries:
   meshery:
-  - apiVersion: v2
```
Comment (leecalcote) [inline charts/index.yaml:4]: These changes should not be included in this PR. Perhaps, you'll need to resync your fork, @shahkushal38?

**Review by leecalcote — CHANGES_REQUESTED** [review]
